index.js

```diff
@@ -1,12 +1,162 @@
 const express = require('express');
 const helmet = require('helmet');
+const knex = require('knex');
+const knexConfig = require('./knexfile');
+
+const db = knex(knexConfig.development);
 
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
-// endpoints here
+// ZOOS TABLE
+
+server.post('/api/zoos', (req, res) => {
+  const zoo = req.body;
+  if (!zoo.name) {
+    res.status(500).json({error: "Please provide a name."});
```

Comment:
Instead if a 500 error code, this might be better tasked to a 400 error code since the error would be a result of user error. The error here would not be on our end as backend developers/client maintenance.